src/lib/ledger_catchup/super_catchup.ml

```diff
@@ -863,6 +874,7 @@ let setup_state_machine_runner ~t ~verifier ~downloader ~logger
                           record trust_system logger peer
                             Actions.(Sent_invalid_proof, None))
                         |> don't_wait_for ) ;
+                    (* TODO should we reject the validation callback? *)
```

Comment:
@nholland94 what do you think about this one?

Reply:
Yes, we should reject here.